### PR TITLE
[MM-9984] Remove WebRTC end user settings

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Remove WebRTC end user settings (account settings) and let the WebRTC functionality toggled on/off at the admin settings (system console) only

#### Ticket Link
Jira ticket: [MM-9984](https://mattermost.atlassian.net/browse/MM-9984)

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Has UI changes
- [x] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates

### CODE REVIEW

File: components/user_settings/advanced/user_settings_advanced.jsx [hunk 1]
```diff
@@ -574,14 +561,11 @@ export default class AdvancedSettingsDisplay extends React.Component {
 }
 
 AdvancedSettingsDisplay.propTypes = {
-    user: PropTypes.object,
     updateSection: PropTypes.func,
-    updateTab: PropTypes.func,
```
Comment (saturninoabril) [inline components/user_settings/advanced/user_settings_advanced.jsx]: Not related to this PR but just remove this unused props. Will submit another PR to do the same with other user settings.

File: components/webrtc/notification/webrtc_notification.jsx [hunk 2]
```diff
@@ -84,35 +83,24 @@ export default class WebrtcNotification extends React.Component {
...
+                });
+                this.stopRinging();
+            } else if (Utils.isUserMediaAvailable()) {
+                WebrtcStore.setVideoCallWith(userId);
+                this.setState({
+                    userCalling: UserStore.getProfile(userId),
+                });
             } else {
                 WebSocketClient.sendMessage('webrtc', {
-                    action: WebrtcActionTypes.DISABLED,
```
Comment (lieut-data) [inline components/webrtc/notification/webrtc_notification.jsx]: Is there any use case for retaining the `DISABLED` message? I guess if it's disabled at the server level, it's not likely to trigger a message in the first place?

Comment (saturninoabril) [reply components/webrtc/notification/webrtc_notification.jsx]: Yeah, I think you're right. I'll update this.

Comment (saturninoabril) [reply components/webrtc/notification/webrtc_notification.jsx]: Done update.

**Pull request merged**